src/citrine/informatics/predictor_evaluation_metrics.py

```diff
@@ -25,23 +25,15 @@ class PredictorEvaluationMetric(PolymorphicSerializable["PredictorEvaluationMetr
     Abstract type that returns the proper type given a serialized dict.
     """
 
-    def _attrs(self) -> List[str]:
-        return ["typ"]
-
     def __hash__(self):
-        return hash(self.__getattribute__("typ"))
+        return hash(self.typ)
```

Comment:
Identical behavior with no introspection.  Necessary because PredictorEvaluationMetrics get stored in a set.

It seems like this should maybe be `hash(repr(self))` instead, since it seems like you should be able to have non-collision between `CoverageProbability`s with different levels, but that would technically be a breaking change.